packages/tools/tests/test/visualization/config.json

```diff
@@ -2068,6 +2068,13 @@
             "title": "Shadows in RHS mode",
             "playgroundId": "#RO51RT#14",
             "referenceImage": "shadows-rhs.png"
+        },
+        {
+            "title": "3D Procedural Textures",
+            "playgroundId": "#AYZY4Q#6",
+            "renderCount": 5,
+            "referenceImage": "procedural-textures-3d.png",
+            "excludedEngines": ["webgl1"]

```

Comment:
Can you add "webgpu" in the excluded engines? It needs a special flag and we can't easily add it on the CI. Thanks!

Reply:
I still had this open so I added webgpu to the config.